Vic2ToHoI4/Source/HOI4World/HoI4Focus.cpp

```diff
@@ -285,6 +291,10 @@ std::ostream& operator << (std::ostream& output, const HoI4Focus& focus)
 	{
 		output << "		continue_if_invalid = " << focus.continueIfInvalid << "\n";
 	}
+	if (!focus.selectEffect.empty())
```

Comment:
This whole function should not be in this file at all, but in HoIOutputter somewhere along with other templates.